cache/restore_test.go

```diff
@@ -161,34 +161,40 @@ func Test_evaluateKeys(t *testing.T) {
 
 func Test_exposeCacheHit(t *testing.T) {
 	tests := []struct {
-		name string
+		name          string
+		evaluatedKeys []string
 		downloadResult
 		wantEnvs []string
 		wantErr  bool
 	}{
 		{
 			name:           "no cache hit",
+			evaluatedKeys:  []string{"my-cache-key"},
 			downloadResult: downloadResult{},
 			wantEnvs:       []string{},
 			wantErr:        false,
 		},
 		{
-			name: "exact cache hit",
+			name:          "exact cache hit",
+			evaluatedKeys: []string{"my-cache-key"},
 			downloadResult: downloadResult{
 				filePath:   "testdata/dummy_file.txt",
 				matchedKey: "my-cache-key",
 			},
 			wantEnvs: []string{
+				"BITRISE_CACHE_HIT=exact",
 				"BITRISE_CACHE_HIT__my-cache-key=9a30a503b2862c51c3c5acd7fbce2f1f784cf4658ccf8e87d5023a90c21c0714",
 			},
 		},
 		{
-			name: "exact cache hit",
```

Comment:
This test case was the exact copy of the previous, not sure what happened here 🤔 . Anyway, I repurposed it for testing the partial match case, this is why the diff looks strange.